CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.1-pre0] - 2022-06-24
```

Comment:
Pre-release versions have a lower precedence than the associated normal version. Are you sure you don't want to make this 1.2.2-pre0?

Reply:
right thats a good point, Ive updated the version to be `1.2.2-pre0`